addon/ActivityStreams.js

```diff
@@ -103,6 +105,20 @@ function ActivityStreams(metadataStore, options = {}) {
 
   this._previewProvider = new PreviewProvider(this._tabTracker, metadataStore, this._experimentProvider);
 
+  this._pageScraper = null;
+
+  // Only create a PageScraper if we are in the experiment
+  if (this._experimentProvider.data.localMetadata) {
```

Comment:
Can we change this so that it inits based solely on whether the pref is enabled and then have a separate check on activity stream startup that checks the experiment provider and sets the pref?